src/Components/test/E2ETest/ServerExecutionTests/ServerSideAppTest.cs

```diff
@@ -161,6 +162,15 @@ public void RendersContinueAfterReconnect()
                 _ => element.Text != currentValue);
         }
 
+        [Fact]
+        public async Task ReconectsAfterPrerenderingContent()
+        {
+            Navigate("/?Component=HelloWorld", noReload: false);
+            await Task.Delay(500);
+            Browser.FindElement(By.TagName("button")).Click();
+            Browser.Equal("Nice to meet you Dan Roth", () => Browser.FindElement(By.Id("message")).Text);
```

Comment:
Hope this is not too killjoy-ish, but I find it a bit awkward embedding our names in the product source like this. Any chance we could use test examples that aren't specific to team members?

Reply:
Hahaha, no problem